src/libraries/System.Text.Json/src/System/Text/Json/Serialization/ReadStack.cs

```diff
@@ -355,5 +355,8 @@ private void SetConstructorArgumentState()
                 }
             }
         }
+
+        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
+        private string DebuggerDisplay => $"Path:{JsonPath()} Current: ConverterStrategy.{Current.JsonTypeInfo?.PropertyInfoForTypeInfo.ConverterStrategy}, {Current.JsonTypeInfo?.Type.Name}";
```

Comment:
Nit: The use of "=", ":", and ": " as key/value separators across the debugger displays looks inconsistent.

Reply:
Similarly, some use ", " between pairs while others use " "